emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBUtil.java

```diff
@@ -279,6 +281,25 @@ public static String getDynamoDBRegion(Configuration conf, String region) {
     return DynamoDBConstants.DEFAULT_AWS_REGION;
   }
 
+  /**
+   *
+   * @param providerClass - class name loaded from conf used as custom credential provider
+   * @return - credential provider loaded via reflection using class name from conf
+   */
+  public static AwsCredentialsProvider loadAwsCredentialsProvider(
+      String providerClass,
+      Configuration conf) {
+    // First try loading class via static create method which is intended aws-java-sdk-v2 behavior
```

Comment:
[minor] better to link this interface from SDK docs

Reply:
sure can update the comment